bears/c_languages/CPPLintBear.py

```diff
@@ -3,6 +3,12 @@
 from coalib.settings.Setting import typed_list
 
 
+def check_invalid_config(use_spaces, indent_size):
+    if not use_spaces or indent_size != 2:
+        raise ValueError("CPPLint doesn't support indent_size != 2 or "
```

Comment:
This error message can be more clear, "CPPLint doesn't support indent size other than 2 or you are not using spaces". Just a suggestion. I get the point of using != because that's what being done in code, but consistency of natural language will make it more readable and comprehensible.